generator/insn_util.go

```diff
@@ -87,6 +87,10 @@ func (v VLEN) Valid() bool {
 	return 64 <= v && v <= 4096 && v&(v-1) == 0
 }
 
+func (v VLEN) Zvkg_validVLEN() bool{
+	return v == 128
```

Comment:
Are you sure about this? The spec says "All of these instructions work on 128-bit element groups comprised of four 32-bit elements." .. which I don't think this means these instructions are only valid when vlen is 128.

Reply:
Any VLEN>=128 should be okay.

Reply:
Sorry I misunderstood, you're right Element Group width should be 128 not the vlen